CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 FEATURES:
 * Add support for `ext_key_usage_oids` in `vault_pki_secret_backend_role` ([#2108](https://github.com/hashicorp/terraform-provider-vault/pull/2108))
 * Adds support to `vault_gcp_auth_backend` for common backend tune parameters ([#1997](https://github.com/terraform-providers/terraform-provider-vault/pull/1997)).
+* Modified `resource_database_secret_backend_connection`'s `max_open_connections` to match 4 as defined elsewhere, including our [documentation](https://developer.hashicorp.com/vault/api-docs/secret/databases/mysql-maria#max_open_connections) ([#2119]([https://github.com/terraform-providers/terraform-provider-vault/pull/](https://github.com/hashicorp/terraform-provider-vault/pull/2119))).
```

Comment:
```suggestion
* Modified `resource_database_secret_backend_connection`'s `max_open_connections` to match 4 as defined elsewhere, including our [documentation](https://developer.hashicorp.com/vault/api-docs/secret/databases/mysql-maria#max_open_connections) ([#2119](https://github.com/hashicorp/terraform-provider-vault/pull/2119)).
```

Reply:
Thanks for this correction, appreciate it.